configuration.yaml

```diff
@@ -1,7 +1,7 @@
 template: staging
 
 validate:
-- url: https://github.com/rancher/partner-charts.git
-  branch: main
+- url: https://github.com/aiyengar2/partner-charts.git
+  branch: migrate_to_v0.2.1_live
```

Comment:
NOTE: do not merge this PR till https://github.com/rancher/partner-charts/pull/94 is merged. This commit is a TEMPORARY commit that will be removed but is currently necessary.